Libraries/TestHelpers.psm1

```diff
@@ -721,6 +721,40 @@ Function Get-LISAv2Tools($XMLSecretFile) {
 	}
 }
 
+Function Get-SSHKey ($XMLSecretFile) {
+	# Download SSHKey when provide a URL
+	$CurrentDirectory = Get-Location
```

Comment:
the env:TEMP is set at LISAv2 beginning for the execution session.  can we use the TEMP folder by default?  and at the end of LISAv2,  force delete the key at end of LISAv2 for a better cleanup and security consideration.